focus-android/app/src/main/res/values/strings.xml

```diff
@@ -693,7 +693,8 @@
     <string name="cookie_banner_exception_panel_description_state_on_for_site">%1$s will clear this site’s cookies and refresh the page. Clearing all cookies may sign you out or empty shopping carts.</string>
 
     <!-- Long text for a detail explanation indicating what will happen if cookie banner handling is off for a site, this is shown as part of the cookie banner panel in the toolbar. The placeholder will be updated with the app name. -->
-    <string name="cookie_banner_exception_panel_description_state_off_for_site">%1$s can try to automatically reject cookie requests.</string>
+    <string name="cookie_banner_exception_panel_description_state_off_for_site" moz:RemovedIn="111" tools:ignore="UnusedResources">%1$s can try to automatically reject cookie requests. If a reject option isn’t available, %1$s may accept all cookies to dismiss the banner.</string>
+    <string name="cookie_banner_exception_panel_description_state_off_for_site2">%1$s can try to automatically reject cookie requests.</string>
```

Comment:
I think we need to duplicate the comment, as the one above will be deleted when  we remove "cookie_banner_exception_panel_description_state_off_for_site".

Reply:
Done